lib/measured/measurable.rb

```diff
@@ -6,7 +6,7 @@ class Measured::Measurable < Numeric
   def initialize(value, unit)
     raise Measured::UnitError, "Unit value cannot be blank" if value.blank?
 
-    @unit = self.class.unit_system.to_unit_name!(unit)
+    @unit = unit.is_a?(Measured::Unit) ? unit : self.class.unit_system.unit_for!(unit)
```

Comment:
I'm going to stick with what I've already done. We'll continue to support symbols/strings in these two methods, but I'd rather lean towards working with instances, so keeping the tests as-is would hopefully encourage people to do that 😃